packages/jaeger-ui/src/components/common/FilteredList/__snapshots__/highlightMatches.test.js.snap

```diff
@@ -3,45 +3,73 @@
 exports[`highlightMatches(query, text) matches contains: been-kat-br[eak]able 1`] = `
 <span>
   <span>
-    been-kat-br
+    <span
```

Comment:
I thought adding Fragment was generally a no-op, how come there's a new `<span>` element added to the DOM?